selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateWorkspaceTest.java

```diff
@@ -71,21 +71,24 @@ public void checkWorkspaceName() {
     newWorkspace.typeWorkspaceName(TOO_SHORT_WORKSPACE_NAME);
     newWorkspace.waitErrorMessage(WS_NAME_TOO_SHORT);
 
-    assertFalse(newWorkspace.isCreateWorkspaceButtonEnabled());
+    newWorkspace.waitCreateWorkspaceButtonDisabled();
 
     newWorkspace.typeWorkspaceName(TOO_LONG_WORKSPACE_NAME);
     newWorkspace.waitErrorMessage(WS_NAME_TOO_LONG);
-    assertFalse(newWorkspace.isCreateWorkspaceButtonEnabled());
+    newWorkspace.waitCreateWorkspaceButtonDisabled();
 
     // type valid names and check that the Create button is enabled
     newWorkspace.typeWorkspaceName(MIN_VALID_WORKSPACE_NAME);
 
-    assertTrue(newWorkspace.isCreateWorkspaceButtonEnabled());
+    newWorkspace.waitCreateWorkspaceButtonEnabled();
```

Comment:
let's not to divide **type** and **wait** commands of the same step by empty row to make it more readable